architecture-decision-records/0014-govuk-notify-service-configuration.md

```diff
@@ -0,0 +1,52 @@
+# 14. GovUK Notify Service Configuration
+
+**Date:** 2021-04-08
+
+## Context
+
+GovUK Notify is used as the email provider for Keycloak: [0012 Keycloak Email Provider](0012-keycloak-email-provider.md)
+
+There are multiple ways that GovUK Notify can be configured for a multi-environment architecture such as TDR uses.
+
+TDR has three environments:
+* integration
+* staging 
+* production
+
+GovUK Notify does not have a specific concept of different environments.
+
+## Decision: Separate GovUK Notify Services for each TDR Environment
```

Comment:
Would you mind splitting these out into separate decision files? If we change our minds later, it makes it much easier to keep track of them and cross-reference them, e.g. "decision 25 supersedes decision 14".